source-kafka/src/kafka.rs

```diff
@@ -28,12 +29,21 @@ pub enum Error {
 }
 
 pub fn consumer_from_config(configuration: &Configuration) -> Result<BaseConsumer, Error> {
-    ClientConfig::new()
-        .set("bootstrap.servers", configuration.brokers())
-        .set("enable.auto.commit", "false")
-        .set("group.id", "source-kafka")
-        .create()
-        .map_err(Error::Config)
+    let mut config = ClientConfig::new();
+
+    config.set("bootstrap.servers", configuration.brokers());
+    config.set("enable.auto.commit", "false");
+    config.set("group.id", "source-kafka");
```

Comment:
1. I do think that a code comment would be helpful, since the reason for that line to exist isn't obvious.
2. I'm not aware of any reason to make them random, but I really wouldn't know. Seems like it could be worth doing a little digging.